internal/app/networkd/pkg/address/dhcp.go

```diff
@@ -223,13 +225,29 @@ func (d *DHCP) discover() (*dhcpv4.DHCPv4, error) {
 	// nolint: errcheck
 	defer cli.Close()
 
-	lease, err := cli.Request(context.Background(), mods...)
+	lease, err := cli.Request(ctx, mods...)
 	if err != nil {
 		// TODO: Make this a well defined error so we can make it not fatal
 		log.Printf("failed dhcp request for %q: %v", d.NetIf.Name, err)
 
 		return nil, err
 	}
 
+	log.Printf("DHCP ACK on %q: %s", d.NetIf.Name, collapseSummary(lease.ACK.Summary()))
```

Comment:
should be single line on console, printed twice on initial discovery, then it goes to networkd log on each discovery